src/serlio/modifiers/RuleAttributes.cpp

```diff
@@ -115,11 +124,15 @@ RuleAttributeSet getRuleAttributes(const std::wstring& ruleFile, const prt::Rule
 
 		RuleAttribute p;
 		p.fqName = attr->getName();
-		p.mayaBriefName = getBriefName(p.fqName);
-		p.mayaFullName = getFullName(p.fqName);
 		p.mayaNiceName = getNiceName(p.fqName);
 		p.mType = attr->getReturnType();
 
+		//make sure maya names are unique
+		const std::wstring mayaBriefName = getBriefName(p.fqName);
+		const std::wstring mayaFullName = getFullName(p.fqName);
+		p.mayaBriefName = mayaBriefName + getDuplicateCountSuffix(mayaBriefName, mayaNameDuplicateCountMap);
+		p.mayaFullName = mayaFullName + getDuplicateCountSuffix(mayaFullName, mayaNameDuplicateCountMap);
```

Comment:
consider hiding the clash resolution in the brief/full name functions

Reply:
fixed